web/webpack.common.config.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+const OUTPUT_PATH = path.resolve(__dirname, 'dist');
+
+module.exports = {
+  entry: './src/main.ts',
+  output: {
+    path: OUTPUT_PATH,
+    filename: 'bundle.js',
+  },
+  resolve: {
+    /**
+       * See https://webpack.js.org/configuration/resolve/#resolve-extensions
+       * 
+       * ".js" included to make some Webpack plugins work.
+       */
+    extensions: ['.ts', '.tsx', '.js'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(ts|tsx)$/,
+        exclude: /node_modules/,
+        use: ['ts-loader'],
+      },
+      {
+        /**
+           * By default, Webpack (rather, style-loader) includes stylesheets
+           * into the JS bundle.
+           * 
+           * ExtractTextPlugin emits them into a separate plain file instead.
+           */
```

Comment:
@mikehenrty actually, with this plugin we're currently keeping CSS _out_ of the JS (bundle) ;)

the import of the CSS file in the main TS file is just to give webpack an entry point for the CSS (this way, no CSS file needs to be specified separately for the build)